src/main/java/com/example/home_work_4/StringTotal.java

```diff
@@ -0,0 +1,393 @@
+package com.example.home_work_4;
+
+import java.util.Scanner;
+
+public class StringTotal {
+
+    public static void main(String[] args) {
+        //String1_1
```

Comment:
Please extract all the code in separate methods, name them according to the codingbat tasks (check the codingbat resource to find out the names)